vignettes/adrs.Rmd

```diff
@@ -314,6 +312,57 @@ adrs <- adrs %>%
   )
 ```
 
+```{r, echo=FALSE}
+dataset_vignette(
+  adrs,
+  display_vars = exprs(USUBJID, AVISIT, PARAMCD, AVALC, ADT, ANL01FL, ANL02FL)
+)
+```
+
+### Select Source Assessments for Parameter derivations
+
+For most parameter derivations the post-baseline overall response assessments up
+to and including first PD are considered.
+```{r}
+ovr <- filter(adrs, PARAMCD == "OVR" & ANL01FL == "Y" & ANL02FL == "Y")
+```
+
+```{r, echo=FALSE}
+dataset_vignette(
+  ovr,
+  display_vars = exprs(USUBJID, AVISIT, AVALC, ADT, RANDDT)
+)
+```
+
+## Define Events
+
+Common events like complete response (CR), partial response (PR), ... are
+predefined in admiralonco (see [Pre-Defined Event
+Objects](../reference/event_objects.html)). Some may need to be adjusted for
+study-specific needs, e.g., minimum time between response and confirmation
+assessment. Here the `keep_source_vars` argument is updated.
```

Comment:
is there a better example we could use here? as this change is very subtle and most users won't easily understand the difference. how about a hypothetical example where there was another possible response value in your study and then you show how to create a new version of `rsp_y` to include this along with `CR` and `PR`? that might then already help people to visualise the iRECIST examples to come where `iCR` and `iPR` could be used.

Reply:
I have added an example for the confirmation period as this is already mentioned in the text.